.gitignore

```diff
@@ -5,15 +5,15 @@ config.tfvars
 id_rsa*
 pull-secret.json
 ssh-key.pem
-tectonic-*.tar.gz
+openshift-installer-*.tar.gz
 tectonic-license.txt
 terraform.tfstate*
 terraform.tfvars
 bin/
 bin_test/
 matchbox/
 /contrib/govcloud/vpn.conf
-tectonic-dev
+openshift-installer
```

Comment:
I think you mean `openshift-installer-dev`, since this line is for folks following the README and unpacking a `*-dev` tarball for testing.  I'd also be fine with ignoring both `openshift-installer` and `openshift-installer-dev`.